paging/common/src/test/kotlin/androidx/paging/PageFetcherSnapshotTest.kt

```diff
@@ -110,14 +108,12 @@ class PageFetcherSnapshotTest {
             )
         )
         advanceUntilIdle()
-        assertThat(fetcherState.newEvents()).isEqualTo(
-            listOf<PageEvent<Int>>(
-                LoadStateUpdate(PREPEND, false, Loading),
-                createPrepend(
-                    pageOffset = -1,
-                    range = 0..0,
-                    startState = NotLoading.Complete
-                )
+        assertThat(fetcherState.newEvents()).containsExactly(
+            LoadStateUpdate<PageEvent<Int>>(PREPEND, false, Loading),
```

Comment:
Shouldn't this be just `LoadStateUpdate<Int>(..)` ?

Reply:
You're right! Updating now.